srcpkgs/efibootmgr/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'efibootmgr'
 pkgname=efibootmgr
-version=17
-revision=6
+version=18
+revision=1
 hostmakedepends="pkg-config"
 makedepends="libefivar-devel popt-devel"
 short_desc="Tool to modify UEFI Firmware Boot Manager Variables"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/rhinstaller/efibootmgr"
 distfiles="https://github.com/rhinstaller/efibootmgr/releases/download/${version}/${pkgname}-${version}.tar.bz2"
 distfiles="https://github.com/rhboot/efibootmgr/archive/${version}.tar.gz"
```

Comment:
which distfiles is correct?

Reply:
completely passed me by. The second one is the correct one, (rhintsaller is the old name for the organization even though the redirect, give me 5 minutes and I will update the homepage as well